data/preset/blog.yaml

```diff
@@ -0,0 +1,181 @@
+      slots:
+        content:
+          shortdef: 'Layout.Box(orientation: vertical)'
+          slots:
+            contents:
+            - shortdef: 'Banner.App(show-subtitle: false)'
+            - type: ContentGroup.ContentGroup
+              properties:
+                halign: fill
+                expand: true
+              slots:
+                arrangement:
+                  type: Arrangement.List
+                  slots:
+                    card: Card.DefaultFamily
+                selection:
+                  type: Selection.AllHighlights
+                  id: highlighted
+                  slots:
+                    filter: 'Filter.Sets(invert: true)'
+                    order: 'Order.PublishedDate(ascending: false)'
+
+- &article-page
+  type: Layout.ArticleStack
+  properties:
+    do-sliding-animation: false
+    allow-navigation: next
+  slots:
+    card: 'Card.KnowledgeDocument(show-titles: false, show-toc: false)'
+    nav-card: 'Card.List(show-synopsis: false)'
+
+- &set-page
+  shortdef: 'Layout.InfiniteScrolling(expand: true)'
+  references:
+    lazy-load: set-articles
+  slots:
+    content:
+      type: Layout.ResponsiveMargins
+      slots:
+        content:
+          shortdef: 'Layout.Box(orientation: vertical)'
+          slots:
+            contents:
+            - shortdef: 'Banner.Set(halign: start)'
+              slots:
+                card: 'Card.Title(halign: center)'
+            - type: ContentGroup.ContentGroup
+              properties:
+                halign: fill
+                expand: true
+              slots:
+                arrangement:
+                  type: Arrangement.List
+                  slots:
+                    card: Card.DefaultFamily
+                selection:
+                  type: Selection.ArticlesForSet
+                  id: set-articles
+                  slots:
+                    order: 'Order.PublishedDate(ascending: false)'
+
+- &search-page
+  shortdef: 'Layout.InfiniteScrolling(expand: true)'
+  references:
+    lazy-load: search-results
+  slots:
+    content:
+      type: Layout.ResponsiveMargins
+      slots:
+        content:
+          shortdef: 'Layout.Box(orientation: vertical)'
+          slots:
+            contents:
+            - shortdef: 'Banner.Search(halign: start)'
+              slots:
+                card: 'Card.Title(halign: center)'
+            - type: ContentGroup.ContentGroup
+              properties:
+                halign: center
+              slots:
+                arrangement:
+                  type: Arrangement.List
+                  slots:
+                    card: Card.List
+                selection:
+                  type: Selection.Search
+                  id: search-results
+                no-results:
+                  shortdef: 'Layout.Box(orientation: vertical)'
+                  slots:
+                    contents:
+                    - type: ContentGroup.NoResultsMessage
```

Comment:
It occurred to me much later after you asked this morning, that you could just stick `ContentGroup.StaticText` here if you wanted to change the text.